pkg/controller/seed-controller-manager/mla/mla.go

```diff
@@ -35,11 +38,22 @@ import (
 const (
 	ControllerName = "kubermatic_mla_controller"
 	mlaFinalizer   = "kubermatic.io/mla"
+	defaultOrgID   = 1
+)
+
+var (
+	// groupToRole map kubermatic groups to grafana roles
+	groupToRole = map[string]models.RoleType{
+		rbac.OwnerGroupNamePrefix:  models.ROLE_ADMIN,
+		rbac.EditorGroupNamePrefix: models.ROLE_EDITOR,
+		rbac.ViewerGroupNamePrefix: models.ROLE_VIEWER,
```

Comment:
are we importing grafana only for those 3 constants? (I don't see any other usage).

Reply:
yep, only for roles

Reply:
oh, yeah, by bad, we need to `get`, but I need to set header, so again, can't use this method, right?

Reply:
wrong thread? :)

Reply:
of course :)